components/text_recognition/browser/text_recognition.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_TEXT_RECOGNITION_BROWSER_TEXT_RECOGNITION_H_
+#define BRAVE_COMPONENTS_TEXT_RECOGNITION_BROWSER_TEXT_RECOGNITION_H_
+
+#include <string>
+#include <vector>
+
+#include "base/component_export.h"
+
+class SkBitmap;
+
+namespace text_recognition {
+
+// Returns recognized texts from |image|.
+std::vector<std::string> COMPONENT_EXPORT(TEXT_RECOGNITION_BROWSER)
```

Comment:
https://source.chromium.org/search?q=%5ECOMPONENT_EXPORT&sq=&ss=chromium
I believe the EXPORT declaration is preferred before the output type in such cases. so it should be:
`COMPONENT_EXPORT(TEXT_RECOGNITION_BROWSER) std::vector<std::string>`

although in class/struct cases it should go after the class/struct keyword.

Reply:
Fixed. Thanks for you great review! 👍🏼